From randy.dunlap@oracle.com  Mon Aug 30 16:56:54 2010
Date: Tue, 24 Aug 2010 14:09:40 -0700
From: Randy Dunlap <randy.dunlap@oracle.com>
To: devel@driverdev.osuosl.org
Cc: gregkh@suse.de, support@caviumnetworks.com,
	Arnaud Lacombe <lacombar@gmail.com>
Subject: Staging: octeon: depends on NETDEVICES
Message-Id: <20100824140940.57369e83.randy.dunlap@oracle.com>

From: Randy Dunlap <randy.dunlap@oracle.com>

OCTEON_ETHERNET should depend on NETDEVICES.

Fixes this kconfig warning:

warning: (NET_DSA && NET && EXPERIMENTAL && NETDEVICES && !S390 || ...
|| OCTEON_ETHERNET && STAGING && !STAGING_EXCLUDE_BUILD && CPU_CAVIUM_OCTEON) selects PHYLIB which has unmet direct dependencies (!S390 && NETDEVICES)

Reported-by: Arnaud Lacombe <lacombar@gmail.com>
Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Cc: support@caviumnetworks.com
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/octeon/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/octeon/Kconfig
+++ b/drivers/staging/octeon/Kconfig
@@ -1,6 +1,6 @@
 config OCTEON_ETHERNET
 	tristate "Cavium Networks Octeon Ethernet support"
-	depends on CPU_CAVIUM_OCTEON
+	depends on CPU_CAVIUM_OCTEON && NETDEVICES
 	select PHYLIB
 	select MDIO_OCTEON
 	help
